source/math/FloatEuclidTransform.ooc

```diff
@@ -97,12 +100,32 @@ FloatEuclidTransform: cover {
 	operator * (value: Float) -> This {
 		This new(this translationX * value, this translationY * value, this rotationX * value, this rotationY * value, this rotationZ * value, (this scaling - 1) * value + 1)
 	}
+	toImageCoordinates: func(zDistance: Float) -> This {
+		correction := Quaternion getTransform(this rotation, zDistance) * FloatPoint2D new(0.0f, 0.0f)
+		This new(
+			this translationX + correction x,
+			this translationY + correction y,
+			this rotationX,
+			this rotationY,
+			this rotationZ,
+			this scaling)
```

Comment:
I normally like compact code, but in this case I wonder if this wouldn't look better with the `)` on its own line. Opinions?